Guess-The-Bollywood-Movie/index.js

```diff
@@ -0,0 +1,63 @@
+const quizDB = [
+    {
```

Comment:
It is still showing me the first answer 
![image](https://user-images.githubusercontent.com/66599363/135971176-0168fa69-2018-4578-aaea-918f6222963c.png)

Reply:
When we click it the answer will be removed.

Reply:
Okay, I'll try to implement it.